broker-cli/LICENSE

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
This file is already present at the root so we don't need to duplicate it.

Reply:
done